src/functionalTest/resources/envelopes/exception-with-evidence.json

```diff
@@ -0,0 +1,35 @@
+{
+  "id": "0191",
+  "case_ref": "",
+  "po_box": "BULKSCAN PO BOX",
+  "jurisdiction": "BULKSCAN",
+  "container": "bulkscan",
+  "classification": "EXCEPTION",
+  "zip_file_name": "zip-file-test.zip",
+  "delivery_date": "1970-01-01T00:00:00.000Z",
+  "opening_date": "1970-01-01T00:00:00.000Z",
+  "documents": [
+    {
+      "file_name": "certificate1.pdf",
+      "control_number": "154565768",
+      "type": "other",
+      "subtype": null,
+      "scanned_at": "2018-01-01T10:30:00.000Z",
+      "uuid": "ee69aee8-1a33-40dd-9af9-d90da1b104ba"
+    }
+  ],
+  "ocr_data": [
```

Comment:
Before creating the case, the orchestrator will ask the transformation service (in this case our sample app) to transform the exception record. Looks like sample app will require `first_name` and `last_name` fields in OCR data to accept the exception record data.

https://github.com/hmcts/bulk-scan-ccd-event-handler-sample-app/blob/master/src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/common/OcrFieldNames.java

Reply:
Added first_name and last_name